import/PopupStackView.qml

```diff
@@ -0,0 +1,195 @@
+     * \sa Stack
+     */
+    property alias busy: stack.busy
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that enters the stack when another item is popped off of it.
+     * \qmlproperty Transition popEnter
+     * \sa Stack
+     */
+    property alias popEnter: stack.popEnter
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that exits the stack when the item is popped off of it.
+     * \qmlproperty Transition popExit
+     * \sa Stack
+     */
+    property alias popExit: stack.popExit
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that enters the stack when the item is pushed onto it.
+     * \qmlproperty Transition pushEnter
+     * \sa Stack
+     */
+    property alias pushEnter: stack.pushEnter
+
+    /*!
+     * \brief This property holds the transition that is applied to the item 
+     * that exits the stack when another item is pushed onto it.
+     * \qmlproperty Transition pushExit
+     * \sa Stack
+     */
+    property alias pushExit: stack.pushExit
+
+    /*!
+     * \brief Signal emitted when an attachment thumbnail is clicked.
+     * The \a index of the PopupAttachment in the PopupAttachmentListModel
+     * of the currently displayed PopupView.
+     */
+    signal attachmentThumbnailClicked(var index)
+
+    clip: true
+
+    implicitWidth: 300 + padding
+
+    implicitHeight: 300 + padding
+
+    padding: 5
+
+    background: Rectangle {
+        color: palette.base
+        border {
+            color: palette.shadow
+            width: 2
+        }
+        radius: 2
+    }
+
+    contentItem: GridLayout {
+        columns: 2
+        anchors {
+            fill: parent
+            margins: 5
+        }
+        Button {
+            text: "Previous"
+            onClicked: stack.pop()
+            Layout.alignment: Qt.AlignLeft
+            Layout.fillWidth: true
+            enabled: popupManagers ? stack.depth > 1 : false
+        }
+        Button {
+            text: "Next"
+            onClicked: stack.push(popupViewPage)
+            Layout.alignment: Qt.AlignRight
+            Layout.fillWidth: true
+            enabled: popupManagers ? stack.depth < popupManagers.length : false
+        }
+        Text {
+            Layout.columnSpan: 2
+            Layout.fillWidth: true
+            horizontalAlignment: Text.AlignHCenter
+            text: popupManagers && popupManagers.length > 0 ? `${stack.depth} of ${popupManagers.length}` : ""
+            color: palette.text
+        }
+        StackView {
+            id: stack
+            Layout.columnSpan: 2
+            Layout.fillWidth: true
+            Layout.fillHeight: true
+            initialItem: Component {
+                id: popupViewPage
+                PopupView {
+                    popupManager: popupManagers && popupManagers.length >= StackView.index ? popupManagers[StackView.index] : null
```

Comment:
should `StackView.index` be `stack.index`?

Reply:
No this is an attached property.